### DISCUSSION TITLE
I used the command "jupyter lite build --pyodide" to package the environment, which can load the JupyterLab interface but cannot run Python code.

### DISCUSSION THREAD

**Discussion by JackYinpei** [jupyterlite/jupyterlite#1247] [Q&A] [answered]
As mentioned in the title, I used the command `jupyter lite build --pyodide https://github.com/pyodide/pyodide/releases/download/0.22.1/pyodide-0.22.1.tar.bz2` to create the `dist` folder. 

Then, when I executed `jupyter lite serve --output-dir dist`, the user interface loaded successfully, but I encountered an issue when trying to execute Python code. 

and this error occur
![image](https://github.com/jupyterlite/jupyterlite/assets/49555788/89297d58-a72b-4ce8-898d-6c863fd7dd7b)

here is the detail 
![image](https://github.com/jupyterlite/jupyterlite/assets/49555788/0015a9b4-7d3f-4f94-9bcf-36578f8f8ea4)

I would like to know if there are any configuration options or settings I may have missed. Any advice or suggestions would be greatly appreciated. Thank you in advance.

**Comment by JackYinpei** [comment, marked as answer]
Hello

I asked GPT, and it suggested that the issue might be related to the version. I tried switching to the version "https://github.com/pyodide/pyodide/releases/download/0.24.1/pyodide-0.24.1.tar.bz2," and the previous problem no longer occurred. However, a new issue has arisen, as shown in the screenshot below.

![image](https://github.com/jupyterlite/jupyterlite/assets/49555788/73c5a1e2-c401-4d20-9944-644e7f80b0da)

I'm in a situation without the internet, and I want all the dependencies for the environment to be downloaded from my server instead of a CDN.

Any advice or suggestions would be greatly appreciated. Thank you in advance.

**Reply by jtpio** [reply]
> I'm in a situation without the internet, and I want all the dependencies for the environment to be downloaded from my server instead of a CDN.

Maybe this guide in the documentation can be helpful: https://jupyterlite.readthedocs.io/en/stable/howto/configure/advanced/offline.html
